src/server/cmd/pachtf/main.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
+	"github.com/pachyderm/pachyderm/v2/src/internal/pachsql"
+	"github.com/pachyderm/pachyderm/v2/src/internal/secrets"
+	"github.com/pachyderm/pachyderm/v2/src/internal/transforms"
+	"github.com/sirupsen/logrus"
+	"golang.org/x/net/context"
+)
+
+const (
+	pfs    = "/pfs"
+	pfsOut = "/pfs/out"
+)
+
+func main() {
+	ctx := context.Background()
+	log := logrus.StandardLogger()
+	args := os.Args[1:]
+	if len(args) < 1 {
+		log.Fatal("at least 1 argument required")
+	}
+	transformName := args[0]
+	transformArgs := args[1:]
+	switch transformName {
+	case "sql-ingest":
+		if err := sqlIngest(ctx, log, transformArgs); err != nil {
+			log.Fatal(err)
+		}
+	default:
+		log.Fatalf("unrecognized transform name %q", transformName)
+	}
+}
+
+func sqlIngest(ctx context.Context, log *logrus.Logger, args []string) error {
+	const passwordEnvar = "PACHYDERM_SQL_PASSWORD"
+	if len(args) < 2 {
+		return errors.Errorf("must provide db url and format")
+	}
+	urlStr, formatName := args[0], args[1]
+	password, ok := os.LookupEnv(passwordEnvar)
+	if !ok {
+		return errors.Errorf("must set %v", passwordEnvar)
+	}
+	u, err := pachsql.ParseURL(urlStr)
+	if err != nil {
+		return err
+	}
+	inputDir := filepath.FromSlash(pfs + "/in")
```

Comment:
What's the thought behind the "/in" directory? I'm not immediately sure why it would be better (or worse) than having query files at the top level of the input repo.

Reply:
This is the path under /pfs that the queries are written to.  It can't be empty and has to match the pipeline spec.

Reply:
Do you think it would be worth making it configurable? I think having it hard coded would be problematic if there were multiple SQL queries being ingressed since there would have to be a separate input repo for each

Reply:
The repo name can be mapped to `in` in the pipeline spec, using the `name` field.  I think hardcoding a value here is preferable to threading through the repo name.

Reply:
sounds good to me 👍